src/main/perl/Fetch/Config.pm

```diff
@@ -40,9 +40,12 @@ sub _config
 
     my @keys = qw(tmp_dir context_url);
     push(@keys, @CFG_KEYS);
-    foreach my $p (@keys) {
+    foreach my $k (@keys) {
+        my $K = uc($k);
```

Comment:
Opinions on $k and $K -- too subtle and easy to miss/typo? Can't say I like $uck either. $ucK might work? Or, tbh I might be tempted to spell out uc($k) every time - it's not saving very much typing/reading to use a separate variable